monitoring/pra/alerts.yaml

```diff
@@ -0,0 +1,79 @@
+x-inputs:
+  - name: namespace
+    type: constant
+    value: zenko
+  - name: kafka_connect_src_job
+    type: config
+    value: artesca-data-dr-source-base-queue-connector-metrics
+  - name: kafka_connect_sink_job
+    type: config
+    value: artesca-data-dr-base-queue-connector-metrics
+  - name: dr_sink_instance
+    type: config
+    value: artesca-data-dr
+  - name: RTO
+    type: constant
+    value:  86400 # 24 hours
+  - name: mongo_jobs
+    type: config
```

Comment:
should be constant

Reply:
=> https://github.com/scality/Zenko/pull/2130#discussion_r1723343402